django/db/models/fields/related.py

```diff
@@ -1273,26 +1281,46 @@ def __init__(self, field, to, related_name=None, limit_choices_to=None,
     # AttributeError
     @cached_property
     def model(self):
-        if not self.field.model:
-            raise AttributeError(
```

Comment:
Just want to make sure this was a conscious deletion and not a bad merge conflict cause it was from the `RelatedObject` removal and not testing I think.